### ISSUE TITLE
BUG: Help Message for `RichTextEditor` cannot be clicked when in `CreationDialog`

### ISSUE THREAD

**Issue by grebaldi** [neos/neos-ui#3318]
### Description

When including a `RichTextEditor` in the node creation dialog, a configured accompanying help message cannot be reached, because the help message button is not clickable.

### Steps to Reproduce

Here's a node type configuration with which the issue can be reproduced:

```yaml
'Vendor.Site:Content.Example':
  superTypes:
    'Neos.Neos:Content': true

  ui:
    label: 'RichTextEditor help message example'
    inspector:
      groups:
        examples:
          label: 'Examples'

  properties:

    codeEditorExample:
      type: string
      ui:
        label: 'CodeEditor Example'
        showInCreationDialog: true
        reloadIfChanged: true
        help:
          message: >
            This is a code editor example.
        inspector:
          group: examples
          editor: 'Neos.Neos/Inspector/Editors/CodeEditor'

    richTextEditorExample:
        type: string
        ui:
          label: 'RichTextEditor Example'
          showInCreationDialog: true
          reloadIfChanged: true
          help:
            message: >
                This is a rich text editor example.
          inspector:
            group: examples
            editor: 'Neos.Neos/Inspector/Editors/RichTextEditor'
```

1. Create a node with a node type like the one described above
2. Try an click on the little help icon below the RichtTextEditor button in the creation dialog

#### Expected behavior

The help message button is clickable and triggers the respective help message to be shown.

#### Actual behavior

https://user-images.githubusercontent.com/2522299/210354872-534fa604-dbd6-4d85-bca5-37c819a034ea.mp4

### Affected Versions

Neos: 7.3

UI: 7.3
